.docker/prod.Dockerfile

```
@@ -30,6 +31,8 @@ RUN chgrp -R ospd-openvas /etc/openvas/ && \
 
 COPY dist/* /ospd-openvas
 
+# allow openvas to access raw sockets
+RUN setcap cap_net_raw=ep /usr/local/sbin/openvas
```

I would recommend using
```suggestion
RUN setcap cap_net_raw+ep /usr/local/sbin/openvas
```
to prevent overriding existing caps.